src/Microsoft.Identity.Web.Sidecar/OpenAPI/Microsoft.Identity.Web.Sidecar.json

```diff
@@ -146,13 +146,20 @@
             "schema": {
               "type": "string"
             }
+          },
+          {
+            "name": "Tenant",
+            "in": "query",
+            "schema": {
+              "type": "string"
+            }
           }
         ],
         "requestBody": {
           "content": {
             "application/json": {
               "schema": {
-                "$ref": "#/components/schemas/DownstreamApiOptions"
+                "$ref": "#/components/schemas/DownstreamApiOptions2"
```

Comment:
Why is there a 2 here, @keegan-caruso ?

Reply:
Yeah, good question, this is how the generator created the spec. Two different input objects reference the same C# object (DownstreamApiOptions). That reference is created twice. Not sure why.